gulpfile.js

```diff
@@ -733,6 +541,110 @@ const copyRootTSConfig = ((cache) =>
     await fs.writeFile(filePath, JSON.stringify(tsconfig, null, 2));
   }))({});
 
+const CLIENT_PACKAGES = [
+  '@neo-one/client',
+  '@neo-one/client-common',
+  '@neo-one/client-core',
+  '@neo-one/developer-tools',
+  '@neo-one/monitor',
+  '@neo-one/utils',
+  '@neo-one/client-switch',
+  '@neo-one/server-http-client',
+  '@neo-one/node-vm',
+  '@neo-one/client-full-common',
+  '@neo-one/node-core',
+];
+
+const CLIENT_FULL_PACKAGES = CLIENT_PACKAGES.concat(['@neo-one/client-full', '@neo-one/client-full-core']);
+
+const CLIENT_BROWSERIFY = {
+  'neo-one-client': CLIENT_PACKAGES,
+  'neo-one-client-full': CLIENT_FULL_PACKAGES,
+};
+
+const webpackBrowserifyConfig = (externals, inputPath, outputPath) => ({
+  entry: inputPath,
+  output: {
+    path: outputPath,
+    filename: 'index.js',
+    libraryTarget: 'commonjs',
+  },
+  node: {
+    fs: 'empty',
+  },
+  externals,
+});
+
+const createPkgJSONMap = () => {
+  const { pkgJSONs } = getPackageJSONs();
+
+  return pkgJSONs.reduce(
+    (acc, pkgJSON) => ({
+      ...acc,
+      [pkgJSON[1].name]: pkgJSON[1],
+    }),
+    {},
+  );
+};
+
+const EXTERNALS_BLACKLIST = new Set(['through', 'wif']);
+
+const getClientPkgDependencies = (pkgJSONMap, pkgs) => {
+  const externals = new Set();
+  pkgs.forEach((pkg) => {
+    const dependencies = pkgJSONMap[pkg].dependencies;
+    Object.entries(dependencies).forEach(([name, version]) => {
+      if (!(name.includes('@neo-one') || name.includes('@types') || EXTERNALS_BLACKLIST.has(name))) {
+        externals.add([name, version]);
+      }
+    });
+  });
+
+  return [...externals].reduce(
+    (acc, dependency) => ({
+      ...acc,
+      [dependency[0]]: dependency[1],
+    }),
+    {},
+  );
+};
+
+const bundleBrowserify = ((cache) =>
+  memoizeTask(cache, async function bundleBrowserify(format) {
+    if (format.dist === 'neo-one') {
+      const pkgJSONMap = createPkgJSONMap();
+
+      Object.entries(CLIENT_BROWSERIFY).map(([pkgName, dependencies]) => {
+        const externals = getClientPkgDependencies(pkgJSONMap, dependencies);
+        const inputPath = path.resolve(__dirname, getDest(format), pkgName);
+        const outputPath = path.resolve(__dirname, getDest(format), `${pkgName}-browserify`);
+
+        webpack(webpackBrowserifyConfig(Object.keys(externals), inputPath, outputPath), (err, stats) => {
+          if (err) {
+            throw err;
+          } else if (stats.hasErrors()) {
+            throw new Error('Webpack bundling failed.');
+          }
+
+          const pkgJSON = fs.readJSONSync(path.resolve(inputPath, 'package.json'));
+          fs.writeJSONSync(
```

Comment:
I think you need to make sure all of the other files that are expected in the package are present (i.e. the other ones that are copied over or generated for the other packages)